src/components/Footer/FooterView.js

```diff
@@ -2,6 +2,6 @@ import React from 'react';
 import { Footer } from 'bento-components';
 import FooterData from '../../bento/globalFooterData';
 
-const ICDCFooter = () => <><Footer data={FooterData} /></>;
+const ICDCFooter = () => <><Footer data={FooterData} background={FooterData.bg} /></>;
```

Comment:
@jonkiky when you get a chance can you add the props as story in bento components

Reply:
Sure, Will do